.github/workflows/ci.yml

```diff
@@ -136,3 +139,26 @@ jobs:
     - name: Evaluate test results vs Windows file
       run: |
         python $SAMNTDIR/test/compare_lk_test_output.py ${SAMNTDIR}/test_results_win64.csv ${SAMNTDIR}/build/test_results_linux64.csv true
+
+    - name: Upload Artifacts
+      uses: actions/upload-artifact@v2
+      with:
+          name: Test Results Linux
+          path: ${{env.SAMNTDIR}}/build/test_results_linux64.csv
+
+    - name: Set up Python ${{ matrix.python-version }}
```

Comment:
Given that the evaluate test results steps also Python, should this be moved earlier in the test file?

Reply:
I'm happy to move it